oxenstored/connection.ml

```diff
@@ -185,9 +185,9 @@ let anon_id_next = ref 1
 let get_domstr con =
   match con.dom with
   | None ->
-      "A" ^ string_of_int con.anonid
+      Printf.sprintf "DOM0 (ANON CONN #%s)" (string_of_int con.anonid)
   | Some dom ->
-      "D" ^ string_of_int (Domain.get_id dom)
+      "DOMID" ^ string_of_int (Domain.get_id dom)
```

Comment:
would use `sprintf "..%d.."` in both cases and use `"DOM%d"` or `"Dom%d"`

Reply:
Done